packages/core/src/screens/main/components/date.tsx

```
@@ -16,11 +16,11 @@ type Document =
 /**
  * We should always use the GMT date, and then format it based on the user's timezone.
  */
-export const Date = ({ row, column }: CellContext<{ document: Document }, string>) => {
+export function Date({ row, column }: CellContext<{ document: Document }, string>) {
```

Biome (2.3.13)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)